### PR TITLE
Merge pull request #16 from anewage/dev

### PR DESCRIPTION
**Pull request by parinaznasr**
Add state changes

### CODE REVIEW

File: components/Chord/ChordDiagram.vue [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+            {{ ancestorPath(arc) }}
+          </title>
+        </path>
+      </g>
+    </transition-group>
+    <!--TEXT-->
+    <transition-group
+      id="SunburstLables"
+      tag="g"
+      name="fade"
```
Comment (anewage) [inline components/Chord/ChordDiagram.vue]: Sample

**Review by anewage — CHANGES_REQUESTED** [review]
Please change something

File: layouts/default.vue [hunk 2]
```diff
@@ -123,6 +123,11 @@ export default {
           icon: 'chrome_reader_mode',
           title: 'Shuffler',
           to: '/shuffler'
+        },
+        {
+          icon: 'chrome_reader_mode',
+          title: 'Triage',
+          to: '/Triage'
```
Comment (anewage) [inline layouts/default.vue]: Please make this lowercase:
`to: ‘/triage’`

File: pages/triage.vue [hunk 3]
```diff
@@ -0,0 +1,165 @@
+<template>
+  <v-container fill-height fluid>
+    <v-layout row justify-center>
+      <v-flex xs8>
+        <div :id="charts.chordDiagram.id">
+          <chord-diagram
+            :a="mySampleData"
```
Comment (anewage) [inline pages/triage.vue]: What is `a`? Please change this to a more meaningful name.

File: pages/triage.vue [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+      }
+    }
+  },
+  mounted() {
+    this.resize()
+    window.addEventListener('resize', this.resize)
+    // this.$nextTick(() => {
+    //   debugger
+    //
+    // })
```
Comment (anewage) [inline pages/triage.vue]: Please remove the comments and keep the code clean :)

File: components/Chord/ChordDiagram.vue [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+    <transition-group
+      id="Bubbles"
+      tag="g"
+      name="fade"
+      :duration="transitionDuration"
+    >
+      <g v-for="(item, index) in users" :key="index">
+        <circle
+          :cx="radius"
+          :cy="radius"
```
Comment (anewage) [inline components/Chord/ChordDiagram.vue]: Cy and cx are the coordinates of the center of a circle. Please read this carefully: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/circle

Comment (anewage) [inline components/Chord/ChordDiagram.vue]: @parinaznasr Why are you only giving `radius` to the circle’s `cx` and `cy`? It should be something like this:
```
:cx=“item.someThingThatHasACoordinateRelativeToX”
:cy=“item.someThingThatHasACoordinateRelativeToY”

```

**Review by anewage — CHANGES_REQUESTED** [review]
Overall code looks good. Please change the characteristics of text data and circles and bind them to the data element.

File: components/Chord/ChordDiagram.vue (hunk 5 above)
Comment (parinaznasr) [reply components/Chord/ChordDiagram.vue]: It's not finished yet, I'm still working on this part :)

Comment (parinaznasr) [reply components/Chord/ChordDiagram.vue]: I'm looking for a function that assigns different values to cx and cy of each circle to distribute them among a limited area ( the inner circle of the graph)

**Pull request merged**